src/adapter/sources.ts

```diff
@@ -326,28 +326,37 @@ export class SourceContainer {
   // which refer to the same source map will be returned given the location in source map source.
   // This method does not wait for the source map to be loaded.
   currentSiblingUiLocations(uiLocation: UiLocation, inSource?: Source): UiLocation[] {
-    return this._uiLocations(uiLocation).filter(uiLocation => !inSource || uiLocation.source === inSource);
+    return this.allPossibleLocations(uiLocation).filter(uiLocation => !inSource || uiLocation.source === inSource);
   }
 
-  _uiLocations(uiLocation: UiLocation): UiLocation[] {
-    const result: UiLocation[] = [];
-    this._addSourceMapUiLocations(uiLocation, result);
-    result.push(uiLocation);
-    this._addCompiledUiLocations(uiLocation, result);
-    return result;
+  /**
+   * Returns all the possible locations the given original location can map
+   * to, taking into account source maps.
+   */
+  public allPossibleLocations(uiLocation: UiLocation): UiLocation[] {
```

Comment:
Aside from making this public, no functional changes in this file; I was anticipating using some of these methods directly but it turns out that this existing method was what I needed. Left it as the slightly more idiomatic implementation (at least for someone a FP background 😛 )